cypress/integration/analytics.test.js

```diff
@@ -147,47 +147,29 @@ describe('view_item event', () => {
 })
 
 describe('select_item event', () => {
-  beforeEach(() => {
+  it('select_item has the right properties', () => {
     cy.visit(pages.collection, options)
-  })
+    cy.waitForHydration()
+
+    let skuId
 
-  it('add select_item event in data layer', () => {
     cy.getById('product-link')
       .first()
+      .within(() => {
+        cy.getById('buy-button').then(($btn) => {
+          skuId = $btn.attr('data-sku')
+        })
+      })
       .click()
       .then(() => {
-        dataLayerHasEvent('select_item')
```

Comment:
Why do not keep this test?

Reply:
it's already taken into account at line 170